nncf/quantization/layers.py

```diff
@@ -235,15 +247,26 @@ def run_export_quantization(self, x: torch.Tensor):
                                                       input_low,
                                                       input_high)
         if self._export_mode == QuantizerExportMode.ONNX_QUANTIZE_DEQUANTIZE_PAIRS:
-            x, y_scale, y_zero_point = self._prepare_qdq_export_quantization(x)
+            x, y_scale, y_zero_point, per_channel_idx = self._prepare_qdq_export_quantization(x)
+
             if self.per_channel:
-                if torch.allclose(y_scale - y_scale[0], torch.zeros_like(y_scale)) and \
-                        torch.allclose(y_zero_point - y_zero_point[0], torch.zeros_like(y_zero_point)):
-                    y_scale, y_zero_point = y_scale[0], y_zero_point[0]
-                    return ExportQuantizeToONNXQuantDequant.apply(x, y_scale, y_zero_point)
-                raise RuntimeError("PyTorch export to ONNX using QuantizeLinear-DequantizeLinear "
-                                   "doesn't support per channel quantization")
-            return ExportQuantizeToONNXQuantDequant.apply(x, y_scale, y_zero_point)
+                if EXPORT_ONNX_OPSET_VERSION < 13:
+                    if torch.allclose(y_scale - y_scale[0], torch.zeros_like(y_scale)) and \
```

Comment:
Do we still have such checkpoints that need this workaround?

Reply:
Not really, but I thought it would be a waste to just delete this code while it's in perfect working order